requirements.txt

```diff
@@ -5,7 +5,7 @@ PyYAML>=3.11
 psycopg2==2.7.5
 sqlparse==0.2.3
 networkx==1.11
-snowplow-tracker==0.7.2
+minimal-snowplow-tracker==0.0.1
```

Comment:
remove other pinned dependencies because of snowplow-tracker (esp celery)

update setup.py, too